smdebug_rulesconfig/debugger_rules/_utils.py

```diff
@@ -17,18 +28,18 @@ def _get_rule_config(rule_name):
     return rule_config
 
 
-def _get_rule_list(framework, type):
-    rules_list = []
+def _get_rule_list(framework):
+    framework = framework.upper()
+    assert framework in SUPPORTED_FRAMEWORKS
```

Comment:
Add a useful error message listing the supported frameworks.

Reply:
minor suggestion - you can just raise an exception as part of `else` in the clause below. this will make it more clear that each of these cases are mutually exclusive.

Reply:
Done.